src/R/Support/Impl/Help/IntelliSenseRSession.cs

```diff
@@ -36,10 +40,32 @@ public sealed class IntelliSenseRSession : IIntellisenseRSession {
             Session = null;
         }
 
+        public async Task<string> GetFunctionPackageNameAsync(string functionName) {
+            IRSession session = null;
+            string packageName = null;
+
+            if (_workflow.RSession.IsHostRunning) {
+                session = _workflow.RSession;
+            } else if (_coreShell.IsUnitTestEnvironment) {
+                session = Session;
+            }
+
+            if (session != null && session.IsHostRunning) {
+                try {
+                    packageName = await session.EvaluateAsync<string>(
+                        Invariant(
+                            $"if(exists({functionName.ToRStringLiteral()})) getPackageName(environment({functionName})) else NULL"
```

Comment:
Thanks. Yes, I know my variant doesn't work when user overrides well known name i.e. `lm.fit <- function() { }` but it seemed to be a bit exotic. However, I'll try `find`.